common/dal2/src/services/fs.rs

```diff
@@ -90,8 +100,23 @@ impl Accessor for Backend {
     }
 
     async fn write(&self, mut r: Reader, args: &OpWrite) -> Result<usize> {
+        println!("try to write {}", args.path);
```

Comment:
If we want to do print debug, it's better to use `tracing::debug!` instead of `println`.

Reply:
Yep, they should be revmoed.